src/main/resources/assets/bettergrass/lang/zh_tw.json

```diff
@@ -11,11 +11,11 @@
   "bettergrass.crimsonNylium.desc": "也會連接緋紅菌絲石！",
   "bettergrass.warpedNylium.desc": "也會連接扭曲菌絲石！",
   "bettergrass.moreBlocks.desc": "應該在斜坡上連接的類似草地的方塊。",
-  "bettergrass.betterSnow": "Better Snow",
+  "bettergrass.betterSnow": "更好的雪地",
   "bettergrass.betterSnowMode": "Better Snow Mode",
```

Comment:
I've just brought some changes in 7d84a35. Can you please translate these too?

Reply:
Sure! Let me check